app/controllers/skills_controller.rb

```diff
@@ -0,0 +1,4 @@
+class SkillsController < ApplicationController
```

Comment:
is this even used?  It doesnt look like it.  You may want to run the destroy generator to get rid of all the extra files.